packages/server/lib/browsers/cdp_automation.ts

```diff
@@ -144,6 +144,9 @@ export const CdpAutomation = (sendDebuggerCommandFn: SendDebuggerCommand) => {
     })
     .then((result: cdp.Network.GetCookiesResponse) => {
       return normalizeGetCookies(result.cookies)
+      .filter((cookie) => {
+        return !(url.startsWith('http:') && cookie.secure)
+      })
```

Comment:
AFAICT this change became necessary because of https://source.chromium.org/chromium/chromium/src/+/bc6fdb0a984c5e2b807cd7fe442d246d328f5700 - now Chrome is reporting all cookies and not filtering on `Secure` for us